### ISSUE TITLE
Bump Akka 2.6.6

### ISSUE THREAD

**Issue by ignasi35** [akka/akka-sample-cluster-kubernetes-java#21]
With 2.6.6 out this sample needs updating.

We may need to make more changes (not just the version bump) to enabled/demo the SBR.

**Comment by patriknw** [comment]
SBR should be enabled, with config:
```
akka.cluster.downing-provider-class = "akka.cluster.sbr.SplitBrainResolverProvider"
```

**Comment by johanandren** [comment]
Fixed #22
